source/dynamic_data_attachment.md

```diff
@@ -0,0 +1,97 @@
+
+<video controls><source src="./_static/files/dynamic_data_attachment/Connect_Projects_(round_1_survey).mp4" type="video/mp4"></video>
+
+__Step 3:__ Upload and deploy the child project.
+
+<video controls><source src="./_static/files/dynamic_data_attachment/Uploading_and_Deploying_xlsform_(round_2_survey).mp4" type="video/mp4"></video>
+
+__Step 4:__ Go to the child project’s __SETTINGS>Connect Projects__. Click __Select a different project to import data from__. In the dropdown, you will see the projects that can be linked. __Round 1 Survey__ is listed here because it was made available as outlined in __Step 2__. Once a user selects the available project (_Round 1 Survey_ in my case), a unique name is also provided by the system by default (_round_1_survey_ in my case). Users will now need to be cautious and rename the unique name to the name provided in the _xml-external_ question type while designing the child project. Since I had named it _survey_, I have renamed the unique name from _round_1_survey_ to _survey_. Then press __IMPORT__. Once you press the __IMPORT__ button, you will have an option to share or restrict the variables to your child project.
+
+<video controls><source src="./_static/files/dynamic_data_attachment/Connect_Projects_(round_2_survey).mp4" type="video/mp4"></video>
+
+This will now do the magic and link the data from the parent project to the child project. In my case, it will link the data from the _Round 1 Survey_ to the _Round 2 Survey_.
+
+### Projects Owned by Different Users:
+
+Let’s assume a scenario where __userA__ has a parent project which s/he shares with __userB__. Now that this feature is available, __userB__ should be able to upload a child project from his/her user account and link them dynamically with the parent project shared by __userA__. To achieve this at your end, you could follow the steps outlined below:
+
+__Step 1:__ __userA__ should follow the exact steps as outlined in __Step 1__ and __Step 2__ (__Both Projects Owned by Same User__) to _upload_, _deploy_ and _configure_ the parent project.
+
+__Step 2:__ __userA__ should then share the parent project with userB as outlined in [this support article](managing_permissions.md). While sharing, __userA__ should provide minimum access (_View form_, _Edit form_, _View submissions_, and _Add submissions_) to __userB__ else, __userB__ shall now be able to link the project dynamically as expected.
```

Comment:
__Step 2:__ __userA__ should then [share the _parent project_ with __userB__](managing_permissions.md) and grant the minimum permissions of: _View form_, _Edit form_, _View submissions_, and _Add submissions_. __userB__ can now dynamically link their _child project_ to the _parent project_ of __userA__.

Reply:
**check if "Edit form" permissions are required